pyproject.toml

```diff
@@ -42,7 +42,7 @@ dependencies = [
 	"python-fcl >= 0.7",
 	"Rtree ~= 1.0",
 	"rv-ltl ~= 0.1.0a1",
-	"scipy ~= 1.6",
+	"scipy ~= 1.7",
 	"shapely ~= 2.0",
 	"trimesh >3.21.5, <4",
```

Comment:
Was this supposed to have `>=`? Let's change it to use `>=` with the appropriate version in any case so that it's easier to see which versions are acceptable. (With the strict comparison it's not obvious whether post-releases of `3.21.5` are OK for example.)

Reply:
Ah good point